File: google/google.js
function solution(A) {
    // Your solution goes here.
    console.error('Tip: Use console.error() to write debug messages on the output tab.');
    return 0;
  }
  
  // Read from stdin, solve the problem, write answer to stdout.
  const readline = require('readline');
  const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,
    terminal: false
  });
  
  rl.on('line', function(line) {
    const inputs = line.split(' ');
    const A = inputs[0].split(',').map(Number);
  
    process.stdout.write(solution(A).toString());
  });

File: google/google.py
import sys

def solution(A):
  """Your solution goes here."""
  sys.stderr.write('Tip: Use sys.stderr.write() to write debug messages on the output tab.\n')
  pass


def main():
  """Read from stdin, solve the problem, write answer to stdout."""
  input = sys.stdin.readline().split()
  A = [int(x) for x in input[0].split(",")]
  sys.stdout.write(str(solution(A)))


if __name__ == "__main__":
  main()
